frontend/components/ExpandableArea.jsx

```diff
@@ -5,62 +5,38 @@ import shortid from 'shortid';
 // Based on the USWDS Accordion, but only ever has a single
 // item that can be expanded or collapsed
 
-class ExpandableArea extends React.Component {
-  constructor(props) {
-    super(props);
-    this.id = `expandable-area-${shortid.generate()}`;
-
-    this.state = {
-      isExpanded: false,
-    };
-
-    this.toggle = this.toggle.bind(this);
-  }
-
-  toggle() {
-    const { isExpanded } = this.state;
-    this.setState({ isExpanded: !isExpanded });
-  }
-
-  render() {
-    const {
-      bordered, children, render, title,
-    } = this.props;
-    const { isExpanded } = this.state;
-
-    return (
-      <div className={`usa-accordion${bordered ? '-bordered' : ''}`}>
-        <button
-          onClick={this.toggle}
-          className="usa-accordion-button"
-          aria-expanded={isExpanded}
-          aria-controls={this.id}
-          type="button"
-        >
-          {title}
-        </button>
-        <div
-          id={this.id}
-          className="usa-accordion-content"
-          aria-hidden={!isExpanded}
-        >
-          {isExpanded && ((render && render(this.toggle)) || children)}
-        </div>
+function ExpandableArea(props) {
```

Comment:
Was unable to show the `EnvironmentVariableForm` when testing locally.  Looks like the `render` prop from the `ExpandableArea` component is still being used in the `EnvironmentVariables` component.

Reply:
Updated to fix the rendering here. Note that (1) I still plan to update this component to functional in a later PR and (2) this does break the previous functionality of closing the expanded area after submission but it still seems like okay UX in this state.